functions/go/generate-services/README.md

```diff
@@ -0,0 +1,115 @@
+# generate-services
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The generate-services kpt function generates [Config Controller Service resources](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) required for usage of the other Config Controller resources supplied to the function.
+
+<!--mdtogo-->
+
+This removes the requirement that users explicitly enable services, but more importantly, it makes it easier to author kpt packages, because Service resources no longer need to be embedded in every package and there's no need for a standard naming scheme across package providers.
```

Comment:
aah..so this generates resources that enables GCP APIs for the services required by this package. Is this correct ?

Reply:
That is correct